packages/controllers/src/cronjob/CronjobController.test.ts

```diff
@@ -0,0 +1,370 @@
+import {
+  SnapCaveatType,
+  HandlerType,
+  deepClone,
+  TruncatedSnap,
+} from '@metamask/snap-utils';
+import {
+  MOCK_ORIGIN,
+  MOCK_SNAP_ID,
+  getTruncatedSnap,
+} from '@metamask/snap-utils/test-utils';
+import { Duration, inMilliseconds } from '@metamask/utils';
+import { SnapEndowments } from '../snaps';
+import {
+  getRestrictedCronjobControllerMessenger,
+  getRootCronjobControllerMessenger,
+} from '../test-utils';
+import { CronjobController } from './CronjobController';
+
+const MOCK_CRONJOB_PERMISSION = {
```

Comment:
Can you test that this works with the object format too?

Reply:
Yes, the tests are added in the last commit. Permission mock now contains that format too, so it will be tested through all suitable tests.